openshift/machine-config-operator
# Modify MCO image for OpenShift Local

**- What I did**
Create scripts to test a custom image of MCO to run with OpenShift Local (using CRC) similar to hack/cluster-push-prep.sh and hack/cluster-push.sh

**- How to verify it**

Get a local cluster running
```
crc config
crc start
oc login -u kubeadmin https://api.crc.testing:6443
```

[Log 1] See the logs of a MCO pod

`oc logs -n openshift-machine-config-operator machine-config-operator-xxxxxx`


Build your new MCO image `make image`


Push new image to Cluster
```
./hack/local-cluster-push-prep.sh
./hack/local-cluster-push.sh
```

Wait for the cluster to restart

`oc get pods -n openshift-machine-config-operator`

[Log 2] See the logs of a MCO pod

`oc logs -n openshift-machine-config-operator machine-config-operator-xxxxxx`

The Log1 and Log2 should differ in hash value 

**- Description for the changelog**
